pkgs/by-name/si/signal-desktop/signal-desktop-aarch64.nix

```
@@ -1,8 +1,16 @@
 { callPackage }:
 callPackage ./generic.nix { } rec {
   pname = "signal-desktop";
+  version = "7.43.0-1";
```

```suggestion
  version = "7.43.0";
```
or is that ever bigger than 1?

---

I am asking because for example some debian packages from some upstream always have that final -1 appended to their normal version number.